### PR TITLE
fix: Skip canary tests for now

### PR DESCRIPTION
**Pull request by ynotdraw**
Related to https://github.com/CrowdStrike/ember-headless-form/pull/55 / https://github.com/CrowdStrike/ember-headless-form/issues/54, we're hitting the same issue here.

### CODE REVIEW

**Review by nicolechung — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -116,7 +111,8 @@ jobs:
           - ember-4.8
           - ember-release
           - ember-beta
-          - ember-canary
+          # @todo enable this again: https://github.com/CrowdStrike/ember-toucan-core/issues/38
```
[the remark this answers is not in the document]
Comment (ynotdraw) [reply .github/workflows/ci.yml:114]: hah yeah sorry, apparently things weren't formatted properly so I checked those changes in too.  Happy to revert those if they're distracting though

[the remark this answers is not in the document]
Comment (nicolechung) [reply .github/workflows/ci.yml:114]: Nope, was just curious if they were manually formatted or if it was automatic?

[the remark this answers is not in the document]
Comment (danwenzel) [reply .github/workflows/ci.yml:114]: Could we put the formatting changes into a separate commit?

[the remark this answers is not in the document]
Comment (ynotdraw) [reply .github/workflows/ci.yml:114]: These are pretty small, so I'll just revert 😉

[the remark this answers is not in the document]
Comment (ynotdraw) [reply .github/workflows/ci.yml:119]: Fixed @danwenzel

**Review by danwenzel — APPROVED** [review]

**Pull request merged**